<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectMode</class>
 <widget class="QWidget" name="SelectMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>66</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="container" native="true">
       <property name="minimumSize">
        <size>
         <width>538</width>
         <height>229</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>538</width>
         <height>229</height>
        </size>
       </property>
       <widget class="QRadioButton" name="radioOnlineWallet">
        <property name="geometry">
         <rect>
          <x>3</x>
          <y>3</y>
          <width>526</width>
          <height>37</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="text">
         <string>Online Wallet (recommended for most users) - MWC wallet with a
full functionality. Internet connection is required.</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioColdWallet">
        <property name="geometry">
         <rect>
          <x>3</x>
          <y>123</y>
          <width>523</width>
          <height>34</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="text">
         <string>Cold Wallet - Offline MWC wallet with limited functionality
(mainly used by exchanges)</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioOnlineNode">
        <property name="geometry">
         <rect>
          <x>3</x>
          <y>66</y>
          <width>526</width>
          <height>31</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="text">
         <string>Online Node - MWC node that provides blockchain data for
the Cold Wallet</string>
        </property>
       </widget>
       <widget class="control::MwcPushButtonNormal" name="applyButton">
        <property name="geometry">
         <rect>
          <x>189</x>
          <y>189</y>
          <width>157</width>
          <height>40</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>65</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>control::MwcPushButtonNormal</class>
   <extends>QPushButton</extends>
   <header>control/MwcPushButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
